test/ref_ops_test.cpp

```diff
@@ -8607,6 +8607,25 @@ TEST_CASE(topk_test)
     }
 }
 
+TEST_CASE(output_shape_is_tuple_type_test)
```

Comment:
No need to add test here. `ref_ops_test.cpp` is meant for testing "ref" implementation and making sure if it is producing correct results or not. Return is a builtin operator and doesn't have `compute()` defined.